data-generator/src/main/java/cloud/project/datagenerator/rabbitmq/requests/RoomUpdateRequest.java

```diff
@@ -14,7 +14,7 @@
 @Builder
 public class RoomUpdateRequest {
 
-    private String updateType;
+    private DataUpdateType updateType;
```

Comment:
dodawana klasa ale brak importu??

Reply:
nie zkompiluje się

Reply:
jak są w tym samym pakiecie to nie trzeba